css/main.css

```diff
@@ -403,7 +408,8 @@ h1{
 /* Overall flexbox frame stuff */
 .guideFrame{
 	display: flex;
-	width: 100%;
+	min-width: 0;
+	min-height: 0;
```

Comment:
what does having min-width and min-height do here?
min-width / height of 0 I don't think does anything here.

Reply:
see the previous comment about min width and height. The same issue was applied here, where the contentContainer was now the right height but the guideFrame was still as long as the guide content without overriding these values.